tests/integrations/mcs/scheduling/api_test.go

```diff
@@ -513,6 +513,20 @@ func (suite *apiTestSuite) checkFollowerForward(cluster *tests.TestCluster) {
 	defer cancel()
 	follower, err := cluster.JoinAPIServer(ctx)
 	re.NoError(err)
+	defer func() {
+		leader := cluster.GetLeaderServer()
+		cli := leader.GetEtcdClient()
+		testutil.Eventually(re, func() bool {
+			_, err = cli.MemberRemove(context.Background(), follower.GetServer().GetMember().ID())
+			if err != nil {
+				return false
+			}
+			res, err := cli.MemberList(context.Background())
+			return err == nil && len(res.Members) == 1
+		})
+		cluster.DeleteServer(follower.GetConfig().Name)
+		follower.Destroy()
```

Comment:
Since this is an `api suite`, maybe we'd better add the `waitLeader` function as well?

Reply:
We only remove the follower which won't affect the leader.

Reply:
oh, you are right